build-tests.cmd

```diff
@@ -0,0 +1,24 @@
+@echo off
+setlocal
+
+set buildTests=build-tests.log
+echo Running build-tests.cmd %* > %buildTests%
+
+if /I [%1] == [/?] goto Usage
+if /I [%1] == [/help] goto Usage
+
+echo msbuild.exe %~dp0src\tests.builds %* /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=detailed;Append;LogFile=%buildTests% >> %buildTests%
+call msbuild.exe %~dp0src\tests.builds %* /nologo /maxcpucount /v:minimal /clp:Summary /nodeReuse:false /flp:v=detailed;Append;LogFile=%buildTests%
```

Comment:
@chcosta How are you thinking the user would override them? What would the syntax look like? I'm wondering if it might be simpler to just have the user modify the script, especially if it's a temporary change like using diagnostic verbosity.